core/src/main/java/org/apache/iceberg/variants/SerializedObject.java

```diff
@@ -0,0 +1,200 @@
+/*
+ * Licensed to the Apache Software Foundation (ASF) under one
+ * or more contributor license agreements.  See the NOTICE file
+ * distributed with this work for additional information
+ * regarding copyright ownership.  The ASF licenses this file
+ * to you under the Apache License, Version 2.0 (the
+ * "License"); you may not use this file except in compliance
+ * with the License.  You may obtain a copy of the License at
+ *
+ *   http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing,
+ * software distributed under the License is distributed on an
+ * "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+ * KIND, either express or implied.  See the License for the
+ * specific language governing permissions and limitations
+ * under the License.
+ */
+package org.apache.iceberg.variants;
+
+import static org.apache.iceberg.variants.VariantUtil.basicType;
```

Comment:
Seems the build prevents static import and require upper case  static member Can you take a look?

[ant:checkstyle] [ERROR] /Users/aixu/iceberg/core/src/main/java/org/apache/iceberg/variants/SerializedObject.java:21:54: Using a static member import should be avoided - org.apache.iceberg.variants.VariantUtil.basicType. [AvoidStaticImport]

[ant:checkstyle] [ERROR] /Users/aixu/iceberg/core/src/test/java/org/apache/iceberg/variants/TestPrimitiveWrapper.java:29:46: Name 'primitives' must match pattern '^[A-Z][A-Z0-9]*+(_[A-Z0-9]++)*+$'. [ConstantName]